query-engine/connector-test-kit-rs/query-tests-setup/src/logging.rs

```diff
@@ -66,19 +68,38 @@ fn create_env_filter(log_queries: bool, qe_log_level: &str) -> EnvFilter {
 /// specific test outputs for readability.
 ///
 /// It is used from test_macros.
-pub struct PrintWriter;
+pub struct PrintWriter {
```

Comment:
It's clever to use this wrapper to intercept the calls to write and pipe them into the transmitter part of the channel. 🧠